acceptance/framework/consul/helm_cluster.go

```diff
@@ -286,6 +347,51 @@ func (h *HelmCluster) Destroy(t *testing.T) {
 			}
 		}
 
+		// Verify that all deployments have been deleted.
+		deployments, err = h.kubernetesClient.AppsV1().Deployments(h.helmOptions.KubectlOptions.Namespace).List(context.Background(), metav1.ListOptions{LabelSelector: "release=" + h.releaseName})
+		require.NoError(r, err)
+		for _, deployment := range deployments.Items {
+			if strings.Contains(deployment.Name, h.releaseName) {
+				r.Errorf("Found deployment which should have been deleted: %s", deployment.Name)
```

Comment:
Good idea to verify resource deletion... this will hopefully help us diagnose which resource is causing things not to be deleted properly in the future.